### PR TITLE
translate Namespaces to Bahasa Indonesia

### PR DESCRIPTION
**Pull request by zufardhiyaulhaq**
This PR addresses #13929

### CODE REVIEW

**Comment by zufardhiyaulhaq** [comment]
`_index.md` is create in PR #14125 
/assign @girikuncoro

File: content/id/docs/concepts/overview/working-with-objects/namespaces.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: Namespace
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Kubernetes mendukung banyak virtual kluster di dalam satu kluster fisik. Virtual kluster tersebut disebut dengan *namespace*.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/namespaces.md]: nit: virtual kluster -> kluster virtual (indonesian phrase)

File: content/id/docs/concepts/overview/working-with-objects/namespaces.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+```
+NAME          STATUS    AGE
+default       Active    1d
+kube-system   Active    1d
+kube-public   Active    1d
+```
+
+Kubernetes berjalan dengan tiga *namespace* awal:
+
+* `default`, *namespace default* untuk objek yang dibuat tanpa mencantumkan *namespace* pada spesifikasinya
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/namespaces.md]: nit: period at the end of sentence.

File: content/id/docs/concepts/overview/working-with-objects/namespaces.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+* `kube-system`, *namespace* yang digunakan untuk objek yang dibuat oleh sistem Kubernetes
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/namespaces.md]: nit: period at the end of sentence.

File: content/id/docs/concepts/overview/working-with-objects/namespaces.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+* `kube-public`, *namespace* ini dibuat secara otomatis dan dapat diakses oleh semua pengguna (termasuk yang tidak diutentikasi). *Namespace* ini disediakan untuk penggunaan kluster, jika beberapa *resouce* harus terlihat dan dapat dibaca secara publik diseluruh kluster. Aspek publik dari *namespace* ini hanya sebuah konvensi, bukan persyaratan.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/namespaces.md]: nit: diseluruh -> di seluruh

File: content/id/docs/concepts/overview/working-with-objects/namespaces.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
+---
+title: Namespace
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Kubernetes mendukung banyak kluster virtual di dalam satu kluster fisik. kluster virtual tersebut disebut dengan *namespace*.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/namespaces.md]: nit: Kluster virtual (capital) after period

File: content/id/docs/concepts/overview/working-with-objects/namespaces.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+*Namespace* menyediakan ruang untuk nama objek. Nama dari *resource* atau objek harus berbeda di dalam sebuah *namespace*, tetapi boleh sama jika berbeda *namespace*. *Namespace* tidak bisa dibuat di dalam *namespace* lain dan setiap *resource* atau objek Kubernetes hanya dapat berada di dalam satu *namespace*.
+
+*Namespace* merupakan cara yang digunakan untuk memisahkan *resource* kluster untuk beberapa pengguna (dengan [*resource quota*](/docs/concepts/policy/resource-quotas/)).
+
+Dalam versi Kubernetes yang akan datang, objek di dalam satu *namespace* akan mempunyai *access control policies* yang sama secara *default*.
+
+Tidak perlu menggunakan banyak *namespace* hanya untuk memisahkan sedikit perbedaan pada *resource*, seperti perbedaan versi dari perangkat lunak yang sama: gunakan [labels](/docs/user-guide/labels) untuk membedakan *resource* di dalam *namespace* yang sama.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/namespaces.md]: nit: labels -> label

File: content/id/docs/concepts/overview/working-with-objects/namespaces.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+default       Active    1d
+kube-system   Active    1d
+kube-public   Active    1d
+```
+
+Kubernetes berjalan dengan tiga *namespace* awal:
+
+* `default`, *namespace default* untuk objek yang dibuat tanpa mencantumkan *namespace* pada spesifikasinya.
+* `kube-system`, *namespace* yang digunakan untuk objek yang dibuat oleh sistem Kubernetes.
+* `kube-public`, *namespace* ini dibuat secara otomatis dan dapat diakses oleh semua pengguna (termasuk yang tidak diutentikasi). *Namespace* ini disediakan untuk penggunaan kluster, jika beberapa *resouce* harus terlihat dan dapat dibaca secara publik di seluruh kluster. Aspek publik dari *namespace* ini hanya sebuah konvensi, bukan persyaratan.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/namespaces.md]: typo: diutentikasi

File: content/id/docs/concepts/overview/working-with-objects/namespaces.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+Sebagai contoh:
+
+```shell
+kubectl --namespace=<insert-namespace-name-here> run nginx --image=nginx
+kubectl --namespace=<insert-namespace-name-here> get pods
+```
+
+### Mengkonfigurasi preferensi namespace 
+
+Kamu dapat menyimpan konfigurasi *namespace* untuk semua perintah *kubectl* dengan perintah:
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/namespaces.md]: should use backtick for `kubectl` (use backtick for command, fields, yaml, etc.)

**Pull request merged**